### PR TITLE
Initial styling

### PR DESCRIPTION
**Pull request by limavfabio**
This PR adds the initial styling as per the Zeplin design

### CODE REVIEW

File: src/components/Header.jsx [hunk 1]
```diff
@@ -18,6 +20,9 @@ function Header() {
           </NavLink>
         </li>
       </ul>
+      <div className="userOval">
+        <img className="userAvatar" src={userAvatar} alt="" />
+      </div>
```
Comment (ahmetbozaci) [inline src/components/Header.jsx:25]: - [ ] I like your design, everything is cool. But according to the [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/styling.md#:~:text=Your%20final%20design%20should%20match%20Zeplin%27s%20design.), your design should match to original  design. Kindly update it. Font size, font weight positioning of elements, and layouts should be EXACTLY the same as the original design.
- [ ] Also make sure website is fullscreen like in the original design. Maybe you can see white space left and right side on it kindly ignore it. [Design link](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/styling.md#:~:text=Login%20to%20Zeplin%20(hint%3A%20you%20will%20find%20the%20credentials%20in%20the%20sneak%20peek%20of%20the%20project).)
image
___
Original 
image

File: src/components/Book.css [hunk 2]
```diff
@@ -0,0 +1,100 @@
+.bookContainer {
+  display: flex;
```
Comment (ahmetbozaci) [inline src/components/Book.css:2]: - [ ] For DRY CSS kindly combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.
_This is just one example. Please check your CSS file and if some CSS rules are the same please combine them also._
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and CSS best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)

File: src/components/Book.css [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  font-size: 1.5em;
+  font-weight: 600;
+  color: #4395fc;
+}
+
+.percentage {
+  display: flex;
+  flex-direction: column;
+  gap: 6px;
+  justify-content: center;
```
Comment (ahmetbozaci) [inline src/components/Book.css:74]: - [ ] For DRY CSS kindly combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.
_This is just one example. Please check your CSS file and if some CSS rules are the same please combine them also._
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and CSS best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)

File: src/components/AddBookForm.css [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  margin: 0.8em 0;
+  color: rgb(87, 87, 87);
+}
+
+form button {
+  background-color: #4395fc;
+  border: none;
+  border-radius: 5px;
+  color: white;
+  padding: 10px 20px;
```
Comment (ahmetbozaci) [inline src/components/AddBookForm.css:32]: - [ ] For DRY CSS kindly combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.
_This is just one example. Please check your CSS file and if some CSS rules are the same please combine them also._
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and CSS best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 @limavfabio  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Highlights 💯
✔️ Your git-flow is correct. 
✔️ You set up linter files and no errors. 
✔️ Your pull request is very professional. It has a meaningful title and summary. Keep it up.
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: src/components/AddBookForm.css (hunk 4 above)
Comment (limavfabio) [reply src/components/AddBookForm.css:32]: Thank you for your time and for the tips, I'm going to apply them

**Review by BertrandConxy — APPROVED** [review]
Hi @limavfabio, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪 

- No linter errors
- project requirements are met
- project works fine
- good PR description
- correct flow is used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**